docs/source/guides/writer/chapters/parameters.rst

```diff
@@ -232,7 +232,7 @@ by `Replay` to replace the current run `Varianter`_ with the one
 loaded from the replayed job. The workflow with ``ignore_new_data`` could
 look like this::
 
-   avocado run --replay latest -m example.yaml
+   avocado replay latest -m example.yaml
```

Comment:
The `replay` command actually lacks the option to override the varianter input used.  We have to consider if this is something to be re-implemented or not.  But, in any case, this example is no longer relevant.

Reply:
oh, you are right. I will remove it from documentation a will create issue about it, for future.